modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CsharpNetcoreFunctionsServerCodegen.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
I don't think this line is needed: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license